.env.sample

```diff
@@ -1,25 +1,7 @@
 LOG_LEVEL=debug
 LOG_FORMAT=text
 
-HTTP_PORT=8080
```

Comment:
IMO it is better to keep all setting in the ENV instead of modifying the docker-composer file.  Otherwise it wouldn't be possible to run qkm only with OIDC or without TLS etc....